src/ngx_pagespeed.cc

```diff
@@ -749,6 +749,11 @@ ngx_command_t ps_commands[] = {
   ngx_null_command
 };
 
+bool ps_disabled(ps_srv_conf_t* cfg_s) {
```

Comment:
Yes it is.  The enclosing namespace starts on line 665, just before the definition of `struct ps_main_conf_t` above, and ends on line 1287, just after the definition of `ps_determine_port()`.